examples/factory-contract-global/src/lib.rs

```diff
@@ -25,7 +25,7 @@ impl GlobalFactoryContract {
         let code_hash = env::sha256_array(&code);
         self.global_contracts_registered_by_code_hash.insert(name.clone(), code_hash);
 
-        Promise::new(env::current_account_id()).deploy_global_contract(code)
+        Promise::new(env::current_account_id()).publish_contract_by_hash(code.into())
```

Comment:
I don't think you need `.into()` or explicit `CryptoHash::from()` in this file anymore (I believe it is a merge conflict artifact).